analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -3,7 +3,8 @@
           AfterTargets="ResolveCodeAnalysisRuleSet"
           BeforeTargets="CoreCompile">
     <PropertyGroup>
-      <SonarAnalyzerRegressionTestRunning Condition="Exists('$(MSBuildStartupDirectory)\binaries\SonarAnalyzer.dll')">true</SonarAnalyzerRegressionTestRunning>
+      <BinariesFolder>$(MSBuildStartupDirectory)\..\packaging\binaries</BinariesFolder>
+      <SonarAnalyzerRegressionTestRunning Condition="Exists('$(BinariesFolder)\SonarAnalyzer.dll')">true</SonarAnalyzerRegressionTestRunning>
```

Comment:
_Not in this PR:_ Let's remove `SonarAnalyzerRegressionTestRunning` completely. 
We can update line 40 like this:
```
<Error Text="Could not find '$(BinariesFolder)\SonarAnalyzer.dll'" Condition="Exists('$(BinariesFolder)\SonarAnalyzer.dll')" />
```
and move that line right after this `<PropertyGroup>`

Reply:
Done in #5154